APCAppCore/APCAppCore/Library/Scheduler/APCScheduleDebugPrinter.m

```diff
@@ -249,6 +264,23 @@ + (NSString *) stringFromDate: (NSDate *) date
     return result;
 }
 
+- (NSString *) stringWithMillisecondsFromDate: (NSDate *) date
```

Comment:
Yes.  During the above printouts, I use this millisecond timestamp in the header so I can uniquely identify the printouts, and search for them reliably.  I put it in the debugPrinter class to make it easier to use the same date-formatting as other printouts emitted by the debugPrinter.